WindowBoard.h

```diff
@@ -1,15 +1,18 @@
 /*
  * Copyright (c) 2015 Markus Himmel
+ * Copyright 2022, Harshit Sharma <harshits908@gmail.com>
  * This file is distributed under the terms of the MIT license
  */
 
 #ifndef WINDOW_BOARD_H
 #define WINDOW_BOARD_H
 
 #include "GameBoard.h"
+#include "HighscoreWindow.h"
```

Comment:
Why is it needed?

Reply:
Thanks for your reviews, @korli, I've added your suggestions to the pull request.
I moved this #include to `WindowBoard.cpp`.